test/sequential/test-http2-ping-flood.js

```diff
@@ -15,13 +17,15 @@ const kPing = new http2util.PingFrame();
 
 const server = http2.createServer();
 
+let interval;
+
 server.on('stream', common.mustNotCall());
 server.on('session', common.mustCall((session) => {
-  session.on('error', common.expectsError({
-    code: 'ERR_HTTP2_ERROR',
-    message:
-      'Flooding was detected in this HTTP/2 session, and it must be closed'
-  }));
+  session.on('error', (e) => {
```

Comment:
Maybe keep `common.expectsError` and have a second `session.on('error', ...` for `clearInterval`?